client/packages/invoices/src/OutboundShipment/DetailView/Footer/Footer.tsx

```diff
@@ -33,6 +33,7 @@ const createStatusLog = (invoice: OutboundFragment) => {
     [InvoiceNodeStatus.Delivered]: null,
     [InvoiceNodeStatus.Verified]: null,
     [InvoiceNodeStatus.Cancelled]: null,
+    [InvoiceNodeStatus.Received]: null,
```

Comment:
Should this be here, if yep should it be moved between delivered/verified with correct datetime mapped ?

Reply:
Same in prescriptions